flexget/plugins/urlrewrite/torrentleech.py

```diff
@@ -139,7 +139,7 @@ def search(self, task, entry, config=None):
                 log.debug('link phase: %s' % link.contents[0])
                 entry = Entry()
                 # extracts the contents of the <a>titlename/<a> tag
-                entry['title'] = link.contents[0]
+                entry['title'] = unicode(link.contents[0])
```

Comment:
That will not work in Py3 afaik. http://python-future.org/compatible_idioms.html#unicode